shimmingtoolbox/cli/mask.py

```diff
@@ -128,3 +129,65 @@ def threshold(fname_input, output, thr):
     return output
+
+
+@mask_cli.command(context_settings=CONTEXT_SETTINGS,
+                  help="Creates a SCT (SpinalCordToolbox) mask from the input file. Depending on the shape (cylinder,"
+                       " box or Gaussian), a mask is created along z direction. To generate this mask, its center"
+                       " must be specified by the user according the method."
+                       " Return an output nifti file with SCT mask.")
+@click.option('-input', 'fname_input', type=click.Path(), required=True,
+              help="(str): Input nifti file to mask. Must be 3D. Supported extensions are .nii or .nii.gz. Example: "
+                   "data.nii.gz")
+@click.option('-output', type=click.Path(), default=os.path.join(os.curdir, 'mask.nii.gz'),
+              help="(str): Name of output mask. Supported extensions are .nii or .nii.gz. Example: data.nii. (default:"
+                   " (os.curdir, 'mask.nii.gz')).")
+@click.option('-size', default='41',
+              help="(str): Size of the mask in the axial plane, given in pixel (Example: 35) or in millimeter "
+                   "(Example: 35mm). If shape=gaussian, size corresponds to sigma (Example: 45). (default: 41)")
+@click.option('-shape', type=click.Choice(['cylinder', 'box', 'gaussian']), default='cylinder',
+              help="(str): Shape of the mask. (default: cylinder)")
+@click.option('-contrast', type=click.Choice(['t1', 't2', 't2s', 'dwi']), default='t2s',
+              help="(str): Type of image contrast. Only with method=optic. (default: t1)")
+@click.option('-method', type=click.Choice(['optic', 'fitseg']), default='optic',
+              help="(str): Method used for extracting the centerline: "
+                   "- optic: automatic spinal cord detection method"
+                   "- fitseg: fit a regularized centerline on an already-existing cord segmentation. It will "
+                   "interpolate if slices are missing and extrapolate beyond the segmentation boundaries (i.e., every "
+                   "axial slice will exhibit a centerline pixel). (default: optic)")
+@click.option('-centerline_algo', type=click.Choice(['polyfit', 'bspline', 'linear', 'nurbs']), default='bspline',
+              help="(str): Algorithm for centerline fitting. Only relevant with -method fitseg (default: bspline)")
+@click.option('-centerline_smooth', default=30, help="(int): Degree of smoothing for centerline fitting. Only for "
+                                                     "-centerline-algo {bspline, linear}. (default: 30)")
+@click.option('-remove', type=click.IntRange(0, 1), default=1, help="(int): Remove temporary files. (default: 1)")
+@click.option('-verbose', type=click.IntRange(0, 2), default=1,
+              help="(int): Verbose: 0 = nothing, 1 = classic, 2 = expended. (default: 1)")
+def sct(fname_input, output, method, contrast, centerline_algo, centerline_smooth, size, shape, remove, verbose):
```

Comment:
needs to be refactored out of this file.  I will do.

Reply:
needs a more explanatory name; whats sct do? if this was being debugged it would be confusing